Xwt.Gtk/Xwt.GtkBackend/EmbeddedWidgetBackend.cs

```diff
@@ -34,7 +34,7 @@ public EmbeddedWidgetBackend ()
 		{
 		}
 
-		public void SetContent (object nativeWidget)
+		public void SetContent (object nativeWidget, bool reparent)
```

Comment:
The Gtk backend reparents the widget automatically by default: https://github.com/mono/xwt/blob/master/Xwt.Gtk/Xwt.GtkBackend/WidgetBackend.cs#L105
Since we introduce a new API here, I think we should throw an `NotImplementedException` in case `reparent: false`, or add the Gtk specific implementation for the new API.